price_feeder/tasks.py

```diff
@@ -406,6 +433,11 @@ def post_price(self, price_no_precision, info_contracts, re_post, task=None, glo
             Web3.toChecksumAddress(address_medianizer),
             tx_args)
 
+        if 'last_tx' in global_manager:
+            global_manager['tx0'] = global_manager['last_tx']
```

Comment:
tx0 would be like the baseline Tx that we will be trying to replace? In contrast with last tx that would be the lat tx we broadcasted, disregarding if it's a post or re-post? If so, add a comment or document it somewhere

Reply:
shouldn't be an else condition? for the fist time the job runs:
```py
else:
    global_manager['tx0'] = tx_receipt.txid
```